packages/frontend/src/components/project/StageSection.tsx

```diff
@@ -6,13 +6,16 @@ import { ProjectDetailsSection } from './ProjectDetailsSection'
 export interface StageSectionProps {
   title: string
   id: string
-  stage?: StageConfig
+  stage?: StageConfig | 'UnderReview'
 }
 
 export function StageSection({ title, id, stage }: StageSectionProps) {
-  if (stage === undefined || stage.stage === undefined) {
+  if (stage === undefined) {
     return null
   }
+  if (stage === 'UnderReview') {
+    return <span>Under Review 🕵️‍♀️</span>
```

Comment:
This should at minimum be wrapped in a section. Unless we're waiting for the designs, but then leave a TODO comment

Reply:
Yes, this will be a section. I'm taking over this on the FE